spec/services/versioning/file_attachment_revision_updater_spec.rb

```diff
@@ -80,5 +64,15 @@
       expect(next_revision.blob_revision.blob_id).to eq(old_fields[:blob_id])
       expect(next_revision.number_of_pages).to eq(old_fields[:number_of_pages])
     end
+
+    it "can accept a blob_revision as an attribute" do
```

Comment:
I think we usually cover this sort of thing with the 'fields' in the other specs, but I can also see why we might want to call it out specifically...